SMLHelper/Interfaces/IPingHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace SMLHelper.V2.Interfaces
+{
+    /// <summary>
+    /// A handler related to PingTypes
+    /// </summary>
+    public interface IPingHandler
+    {
+        /// <summary>
+        /// Registers a ping type for use when creating a beacon
+        /// </summary>
+        /// <param name="pingName">The name of the new ping type</param>
+        /// <param name="sprite">The sprite that is associated with the ping</param>
+        /// <returns>The newly registered PingType</returns>
+        PingType RegisterNewPingType(string pingName, UnityEngine.Sprite sprite);
```

Comment:
If all `UnityEngine.Sprite`s are simply going to be converted into `Atlas.Sprite`s, then why bother offering it in the first place?

Could we instead just use `Atlas.Sprite` for Subnautica and `UnityEngine.Sprite` for Below Zero?